server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1265,14 +1265,21 @@ public void run() {
 
     context.getTableManager().addObserver(this);
 
-    Thread statusThread = Threads.createThread("Status Thread", new StatusThread());
+    // TODO KEVIN RATHBUN updating the Manager state seems like a critical function. However, the
+    // thread already handles, waits, and continues in the case of any Exception, so critical or
+    // non critical doesn't make a difference here.
+    Thread statusThread = Threads.createCriticalThread("Status Thread", new StatusThread());
     statusThread.start();
 
-    Threads.createThread("Migration Cleanup Thread", new MigrationCleanupThread()).start();
+    // TODO KEVIN RATHBUN migration cleanup may be a critical function of the manager, but the
+    // thread will already handle, wait, and continue in the case of any Exception, so critical
+    // or non critical doesn't make a difference here.
+    Threads.createCriticalThread("Migration Cleanup Thread", new MigrationCleanupThread()).start();
```

Comment:
migration cleanup may be a critical function of the manager, but the thread will already handle, wait, and continue in the case of any Exception, so critical or non critical doesn't make a difference here.